entgql/schema.go

```diff
@@ -178,6 +182,11 @@ func (e *schemaGenerator) buildTypes(g *gen.Graph, s *ast.Schema) error {
 						return err
 					}
 					if def != nil {
+						if s.Types[def.Name] != nil {
+							log.Println(fmt.Sprintf("entgql: the %s.%s field uses the previously defined GQL type: %s",
+								node.Name, f.Name, def.Name))
```

Comment:
Why this logging is needed? I try to follow the unix way and avoid printing logs in non-verbose mode. It also feels like a common thing and I wouldn't want to see it in every "go generate" execution.

Reply:
@a8m: This log is a warning for users in case they have a mistake when having two Enum with the same default name. For example `Todo.status` and `Category.status` will generate same enum `Status`. Possible fix is using the `entgql.Type()` annotation in one of them.

Reply:
I think since GraphQL has a global schema, the generated enum for `Todo.status` should be `TodoStatus` and not `Status`. Also, I don't like the idea of printing in codegen time. I prefer to either fail or ignore this if "it's working". Ent today does not print in codegen and I think also in ent/contrib, so I'd avoid adding this new behavior.